docs/technical-design/launch-darkly-testing-approach.md

```diff
@@ -1,15 +1,149 @@
+These implementations are heavily relent on the data and generated Types in `flags.ts` located in `app-web/src/common-code/featureFlags`. This is done for several reasons:
+- Centralize feature flag lists so when adding new flags it only needs to be done once.
+- Auto generated Types based on the feature flag list help removes the need to update types manually and throughout the codebase.
+- Autocompletion of flag parameters to prevent mistakes and invalid flags when using the implementations.
+
+### Prerequisites for local testing
+Before testing locally, make sure to have the following prerequisites.
+
+- **Launch Darkly API key**: A valid Launch Darkly API key will need to be in your `.envrc.local`, see code block below. Cypress will be making the actual request to Launch Darkly, we will just be intercepting the response. If we had an invalid key here, the request to would return a 404 before we could intercept the response. **This will cause the integration to break in testing.**
```

Comment:
```suggestion
- **Launch Darkly API key**: A valid Launch Darkly API key will need to be in your `.envrc.local`, see code block below. Cypress will be making the actual request to Launch Darkly, we will just be intercepting the response. If we had an invalid key here, the request would return a 404 before we could intercept the response, **which would cause the integration to break in testing.**
```